core/services/feeds/service_test.go

```diff
@@ -981,6 +989,148 @@ func Test_Service_ProposeJob_OCR2(t *testing.T) {
 	}
 }
 
+func Test_Service_ProposeJob_Bootstrap(t *testing.T) {
+	t.Parallel()
+
+	var (
+		id         = int64(1)
+		remoteUUID = uuid.NewV4()
+		args       = &feeds.ProposeJobArgs{
+			FeedsManagerID: 1,
+			RemoteUUID:     remoteUUID,
+			Spec:           BootstrapTemplateSpec,
+			Version:        1,
+		}
+		jp = feeds.JobProposal{
+			FeedsManagerID: 1,
+			RemoteUUID:     remoteUUID,
+			Status:         feeds.JobProposalStatusPending,
+		}
+		spec = feeds.JobProposalSpec{
+			Definition:    BootstrapTemplateSpec,
+			Status:        feeds.SpecStatusPending,
+			Version:       args.Version,
+			JobProposalID: id,
+		}
+		httpTimeout = models.MustMakeDuration(1 * time.Second)
+	)
+
+	testCases := []struct {
+		name    string
+		args    *feeds.ProposeJobArgs
+		before  func(svc *TestService)
+		wantID  int64
+		wantErr string
+	}{
+		{
+			name: "Create success",
```

Comment:
I wonder if it makes sense to combine all of these tests into the main one (OCR2, Bootstrap, and the original Fluxmonitor one) as an additional test case, since these are being executed as switch cases off of one function and the rest of the other tests are identical without any changes anyway.